h/static/scripts/controllers/lozenge-controller.js

```diff
@@ -20,9 +21,21 @@ class LozengeController extends Controller {
   constructor(containerEl, options) {
     super(containerEl, options);
     var lozengeEl = document.createElement('div');
+    var lozengeMarkup = escapeHtml(options.content);
```

Comment:
Since you are escaping the markup when generating the HTML, you don't need to do it here as well. In fact, that could possibly result in the wrong output if `getLozengeFacetNameAndValue` doesn't expect escaped input.